mosip-acceptance-tests/ivv-orchestrator/pom.xml

```diff
@@ -1,280 +1,266 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<parent>
-		<groupId>io.mosip.testrig.dslrig.ivv</groupId>
-		<artifactId>dslrig-ivv-parent</artifactId>
-		<version>1.2.1-java21-SNAPSHOT</version>
-	</parent>
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-	<groupId>io.mosip.testrig.dslrig.ivv.orchestrator</groupId>
-	<artifactId>dslrig-ivv-orchestrator</artifactId>
-	<name>mosip-dsl-test</name>
-	<version>1.2.1-java21-SNAPSHOT</version>
-	<properties>
-		<suiteXmlFile>testng.xml</suiteXmlFile>
-		<maven.sonar.plugin.version>3.7.0.1746</maven.sonar.plugin.version>
-	</properties>
+    <parent>
+        <groupId>io.mosip.testrig.dslrig.ivv</groupId>
+        <artifactId>dslrig-ivv-parent</artifactId>
+        <version>1.2.1-java21-SNAPSHOT</version>
```

Comment:
In develop branch we should not use "java21" in version, it should be 1.2.1-SNAPSHOT